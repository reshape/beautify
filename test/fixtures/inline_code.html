<div>
        <p><a href="/link">link</a> <code>foo</code></p>
</div>
