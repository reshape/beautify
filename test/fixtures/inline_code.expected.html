<div>
  <p><a href="/link">link</a>&nbsp;<code>foo</code></p>
</div>
